user_docs/user_stories/submission/submitting_data.md

```diff
@@ -4,4 +4,10 @@
 A submission of data contains the two main steps, please follow the links for further information:
 
 1. [ Agreement of a Data Processing Contract with GHGA Central](dpc_preparation.md)
-1. [Preparing the Data, including Metadata, for submission](submitter_guide.md)
\ No newline at end of file
+1. [Preparing the Data, including Metadata, for submission](submitter_guide.md)
+
+## Note on Submission Service
+
+!!! info "Limitations on External Submissions"
+
+    GHGA has only recently launched the functionality of the GHGA Data Portal. Our ongoing efforts concentrate on data upload processes and to improve the overall user experience. Since data upload currently includes manual steps by GHGA data stewards, data upload by external users is not yet possible. See also this section in our [FAQ](../../faq.md#how-to-upload-your-data-to-the-ghga-data-portal).
```

Comment:
```suggestion
    GHGA has only recently launched the functionality of the GHGA Data Portal. Our ongoing efforts concentrate on improving the data upload processes and the overall user experience for external submissions. Since data upload currently includes manual steps by GHGA data stewards, data upload by external users is not yet possible. See also this section in our [FAQ](../../faq.md#how-to-upload-your-data-to-the-ghga-data-portal).
```